tools/Performance/DynamoPerformanceTests/ResultsComparer.cs

```diff
@@ -0,0 +1,167 @@
+                var baseLine = new string[] { result.Method, result.Graph, "Base", baseResult.Mean.ToString(), baseResult.Error.ToString(), baseResult.StdDev.ToString() };
+                csv.AppendLine(string.Join(",", baseLine));
+
+                var diffLine = new string[] { "", "", "Diff", result.Mean.ToString(), result.Error.ToString(), result.StdDev.ToString() };
+                csv.AppendLine(string.Join(",", diffLine));
+
+                var compResult = Comparison[result.Method + result.Graph];
+                var compLine = new string[] { "", "", "", compResult.Mean.ToString(), compResult.Error.ToString(), compResult.StdDev.ToString() };
+                csv.AppendLine(string.Join(",", compLine));
+            }
+
+            filePath = GetFullPath(filePath);
+            filePath += "/comparison.csv";
+            File.WriteAllText(filePath, csv.ToString());
+        }
+
+        private BenchmarkResult GetBaseResult(BenchmarkResult result)
+        {
+            var key = result.Method + result.Graph;
+            if (BaseResults.ContainsKey(key))
+            {
+                return BaseResults[key];
+            }
+            return null;
+        }
+
+        private Dictionary<string, BenchmarkResult> ImportResultsCSV(string csvPath)
+        {
+            csvPath = GetFullPath(csvPath);
+            TextFieldParser parser = new TextFieldParser(csvPath);
+            parser.HasFieldsEnclosedInQuotes = true;
+            parser.SetDelimiters(",");
+
+            var header = parser.ReadFields();
+            var iMethod = Array.IndexOf(header, "Method");
+            var iGraph = Array.IndexOf(header, "Graph");
+            var iMean = Array.IndexOf(header, "Mean");
+            var iError = Array.IndexOf(header, "Error");
+            var iStdDev = Array.IndexOf(header, "StdDev");
+
+            var benchmarkResults = new Dictionary<string, BenchmarkResult>();
+            while (!parser.EndOfData)
+            {
+                var brLine = parser.ReadFields();
+                var method = brLine[iMethod];
+                var graph = brLine[iGraph];
+                var mean = Convert.ToDouble(Regex.Replace(brLine[iMean], "[^0-9|.|,]", ""), CultureInfo.InvariantCulture);
+                var error = Convert.ToDouble(Regex.Replace(brLine[iError], "[^0-9|.|,]", ""), CultureInfo.InvariantCulture);
+                var stdDev = Convert.ToDouble(Regex.Replace(brLine[iStdDev], "[^0-9|.|,]", ""), CultureInfo.InvariantCulture);
+                benchmarkResults[method+graph] = (
+                    new BenchmarkResult()
+                    {
+                        Method = method,
+                        Graph = graph,
+                        Mean = mean,
+                        Error = error,
+                        StdDev = stdDev,
+                    }
+                );
+            }
+            return benchmarkResults;
+        }
+        
+
+        private string GetFullPath(string path)
```

Comment:
This can probably be moved to the test helper later since it is loosely related to comparer logic